site/docs/v1/tech/apis/_user-send-set-password-email.adoc

```diff
@@ -0,0 +1,8 @@
+[field]#sendSetPasswordEmail# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+If `true, FusionAuth will send the User an email asking them to set their password. The Email Template that is used is configured in the {set_password_field_config_location} [field]#Setup password# field.
++
+If you set this value to `true`, the [field]#password# field is ignored. FusionAuth will set the initial password to a securely generated random string.
++
+If you have also enabled email verification and do not skip verification with the [field]#skipVerification# parameter, only the setup password email will be sent to the user. Setting up the password using the email sent during this user create operation will verify the User's email if it is not already verified.
++
+If the SMTP email configuration is not complete, or disabled, this value is ignored.
```

Comment:
This "field" we are referencing is a reference to the UI configuration I believe, is that correct? Since this is in the API doc, should we further clarify that, or even mention both the field in the Tenant or Application API, as well as the location of the config in the UI?